tests/test_utils_google_directions.py

```diff
@@ -461,13 +567,19 @@ def test_sending_requests_throws_error_if_key_not_found(mocker):
 
 def test_parsing_routes_with_a_good_response(google_directions_api_response, generated_request):
     data = google_directions.parse_routes(google_directions_api_response, generated_request['path_polyline'])
+    assert_semantically_equal(data, {'google_speed': 7.542857142857143, 'google_polyline': 'ahmyHzvYkCvCuCdDcBrB'})
+
+
+def test_parsing_routes_with_a_response_without_traffic_info(google_directions_api_response_without_traffic_info, generated_request, caplog):
+    data = google_directions.parse_routes(google_directions_api_response_without_traffic_info, generated_request['path_polyline'])
     assert_semantically_equal(data, {'google_speed': 3.7183098591549295, 'google_polyline': 'ahmyHzvYkCvCuCdDcBrB'})
+    assert_logging_warning_caught_with_message_containing(caplog, 'duration_in_traffic was not found')
 
 
 def test_parsing_routes_with_multiple_legs_response(google_directions_api_response_multiple_legs, generated_request):
     data = google_directions.parse_routes(google_directions_api_response_multiple_legs,
                                           generated_request['path_polyline'])
-    assert_semantically_equal(data, {'google_speed': 2.3636363636363638, 'google_polyline': 'ekmyH~nYbBzFblahblah'})
+    assert_semantically_equal(data, {'google_speed': 2.9714285714285715, 'google_polyline': 'ekmyH~nYbBzFblahblah'})
```

Comment:
These are also magic numbers/values - why do we expect `'google_speed': 2.9714285714285715`? I can probably work that out by going and inspecting the  test fixtures used here, but if there is a way to save me from having to do that in order to better understand the test, that would be good.

Reply:
ah ok, i wouldnt say its a magic number, but we could do the (speed) math inside the test and compare those results, would that make this better?

Reply:
Yes, being able to see from the test why we expect that value would be clearer, I think.